pecl install igbinary failed (php7)

> creating libtool
appending configuration tag "CXX" to libtool
configure: creating ./config.status
config.status: creating config.h
running: make
/bin/bash /tmp/pear/temp/pear-build-defaultuserUIoR4M/igbinary-2.0.0/libtool --mode=compile cc  -I. -I/tmp/pear/temp/igbinary -DPHP_ATOM_INC -I/tmp/pear/temp/pear-build-defaultuserUIoR4M/igbinary-2.0.0/include -I/tmp/pear/temp/pear-build-defaultuserUIoR4M/igbinary-2.0.0/main -I/tmp/pear/temp/igbinary -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -c /tmp/pear/temp/igbinary/src/php7/igbinary.c -o src/php7/igbinary.lo
/tmp/pear/temp/pear-build-defaultuserUIoR4M/igbinary-2.0.0/libtool: line 1283: src/php7/igbinary.loT: No such file or directory
mkdir src/php7/.libs
mkdir: cannot create directory 'src/php7/.libs': No such file or directory
Makefile:194: recipe for target 'src/php7/igbinary.lo' failed
make: *** [src/php7/igbinary.lo] Error 1
ERROR: `make' failed

Install succeeded if building from source manually.

--- a/config.m4
+++ b/config.m4
@@ -43,7 +43,7 @@ if test "$PHP_IGBINARY" != "no"; then
     apc_inc_path="$phpincludedir"
     AC_MSG_RESULT([APCU in $apc_inc_path])
     AC_DEFINE(HAVE_APCU_SUPPORT,1,[Whether to enable apcu support])
-  elif test -f "$phpincludedir/ext/apc/apc_serializer.h"; then
+  elif test "$subdir" == src/php5 && test -f "$phpincludedir/ext/apc/apc_serializer.h"; then
     apc_inc_path="$phpincludedir"
     AC_MSG_RESULT([APC in $apc_inc_path])
     AC_DEFINE(HAVE_APC_SUPPORT,1,[Whether to enable apc support])
@@ -82,5 +82,6 @@ if test "$PHP_IGBINARY" != "no"; then
   PHP_INSTALL_HEADERS([ext/igbinary], [igbinary.h $subdir/igbinary.h php_igbinary.h $subdir/php_igbinary.h])
   PHP_NEW_EXTENSION(igbinary, $PHP_IGBINARY_SRC_FILES, $ext_shared,, $PHP_IGBINARY_CFLAGS)
   PHP_ADD_EXTENSION_DEP(igbinary, session, true)
+  PHP_ADD_BUILD_DIR($abs_builddir/$subdir, 1)
   PHP_SUBST(IGBINARY_SHARED_LIBADD)
 fi

